### PR TITLE
entry extraction clasification

### PR DESCRIPTION
**Pull request by aembryonic**

### CODE REVIEW

File: analysis_module/mock_templates.py [hunk 1]
```diff
@@ -0,0 +1,17148 @@
...
+               "203": {
+                  "2302": {
+                     "prediction": 0.4348178874601071,
+                     "threshold": 0.409,
+                     "is_selected": true
+                  },
+                  "2303": {
+                     "prediction": 0.2827691658526723,
+                     "threshold": 0.463,
+                     "is_selected": false
```
Comment (ranjan-stha) [inline analysis_module/mock_templates.py]: true -> True, false -> False, also at other places.

**Review by ranjan-stha — COMMENTED** [review]
also, in core/models.py, the line 460 that has type defined, the max_length should be larger than 20, let's say 50.

**Review by ranjan-stha — APPROVED** [review]

**Review by Mogady — APPROVED** [review]
LGTM

**Comment by aembryonic** [comment]
good for me

**Pull request merged**